variables.tf

```diff
@@ -177,6 +177,32 @@ variable "jira_db_iops" {
   type        = number
 }
 
+variable "jira_db_snapshot_identifier" {
+  description = "The identifier for the DB snapshot to restore from. The snapshot should be in the same AWS Region as the DB instance."
+  default     = null
+  type        = string
+}
+
+variable "jira_rds_master_username" {
+  description = "Master username for the Jira RDS instance."
+  type        = string
+  default     = null
+  validation {
+    condition     = can(regex("^[a-zA-Z_]([a-zA-Z0-9_]).{5,30}$", var.jira_rds_master_username)) || var.jira_rds_master_username == null
+    error_message = "Master username must be set. It must be between 6 and 31 characters long and start with a letter/underscore and contain combination of numbers, letters, and underscore."
+  }
+}
+
+variable "jira_rds_master_password" {
+  description = "Master password for the Jira RDS instance."
```

Comment:
```suggestion
  description = "Master password for the Jira RDS instance. If not provided, the password will be generated randomly."
```

Reply:
I prefer to stick to our pattern to provide all additional information including the default values in comments in the config template file and documentation page. In variable definition we just describe what the variable is for. 

We have this pattern for other variables as well. At the end of the day, variable files is not a reference for users, config template and documentations are.